parser-typechecker/src/Unison/Codebase/SqliteCodebase.hs

```diff
@@ -103,30 +88,6 @@ initWithSetup onCreate =
       codebasePath = makeCodebaseDirPath
     }
 
-data CodebaseStatus
-  = ExistingCodebase
-  | CreatedCodebase
-  deriving (Eq)
-
--- | Open the codebase at the given location, or create it if one doesn't already exist.
-withOpenOrCreateCodebase ::
```

Comment:
I was surprised this was now unused, it turns out there's actually a separate `withOpenOrCreateCodebase` in `Unison.Codebase.Init` 🤷🏼‍♂️